modules/ROOT/pages/customize.adoc

```diff
@@ -1,101 +1,177 @@
-** *Name*
-** *Type*
-** *Page*
-** *Publicly available*
 * https://help.salesforce.com/articleView?id=rss_cms_collection.htm&type=5[CMS Collection, role=external, window=blank]
-* *API Carousel*: Displays APIs and API versions that are available to the community from Anypoint Exchange.
-* *API Details Button*: Provides details about an API.
+* *API Carousel*
++
+Displays APIs and API versions that are available to the community from Anypoint Exchange.
++
+* *API Details Button*
++
+Provides details about an API.
++
 * https://help.salesforce.com/articleView?id=community_builder_rich_text_component.htm&type=5[Rich text editor, role=external, window=blank]
 * https://help.salesforce.com/articleView?id=rss_html_editor.htm&type=5[HTML editor, role=external, window=blank]
-
-Components that are typically used in *API details* pages include:
-
-* *API Header*: Specifies the API name and description.
-* *Tabs*: Contains the following components: *API Documentation Viewer*, which displays API Exchange information,
-*API Console*, which renders information from the API RAML specification, and *Feed*, which displays comments,
-discussions, and questions.
-* *API Access Requestor*: Registers a client application and requests access to the API.
-* *Learn More Button*: Provides additional information about the API.
-
-Components that are typically used in *My Application* pages include:
-
-* *Application Listing*: Lists all client applications that the member has registered and can access.
-
-To reorder items on a page, simply drag and drop them.
+* https://help.salesforce.com/articleView?id=rss_setup_nav_menu.htm&type=5[Navigation Menu, role=external, window=blank]
+
+The *Navigation Menu* component provides the menu for navigating the site, with
+settings for *Max Items to Display (2-15)* and *Menu Item Alignment*, and a
+*Navigation Menu* button to edit the links on the navigation menu. The *Home*
+link is always present and is always the first item. Other links can be added,
+removed, and reordered. Each link other than *Home* can be customized with these
+options:
+
+* *Name*
++
+Sets the link name.
++
+* *Type*
++
+Sets the link type, such as *Community Page* or *External URL*.
++
+* *Page*
++
+Specifies community page links from a list of all community pages.
++
+* *Publicly available*
++
+Sets whether the link is available to all page viewers or only logged in
+community members.
+
+Components typically used in *API details* pages include:
+
+* *API Header*
++
+Specifies the API name and description.
++
+* *Tabs*
++
+Contains these components:
++
+** *API Documentation Viewer*
++
+Displays API Exchange information.
++
+** *API Console*
++
+Renders information from the API RAML specification.
++
+** *Feed*
++
+Displays comments, discussions, and questions.
++
+* *API Access Requestor*
++
+Registers a client application and requests access to the API.
++
+* *Learn More Button*
```

Comment:
Is this a component? Button is part of the name?

Reply:
Correct, the component name is "Learn More Button".